### ISSUE TITLE
Don't lint files that have no React in them

### ISSUE THREAD

**Issue by binarykitchen** [DianaSuvorova/eslint-plugin-react-redux#48]
In my project there are many files with React and some others without. But this linter lints them all, even those without React.

Lots of unnecessary errors like 
```
Definition for rule 'react-redux/connect-prefer-named-arguments' was not found. (react-redux/connect-prefer-named-arguments)
```
in non-React files

**Comment by binarykitchen** [comment]
Ah, ignore. The react-redux plugin was missing.
